application/actions/upload.go

```diff
@@ -37,6 +37,16 @@ func uploadHandler(c buffalo.Context) error {
 		}))
 	}
 
+	if f.Size > int64(domain.MaxFileSize) {
+		domain.ErrLogger.Printf("file upload size (%v) greater than max file size (%v)", f.Size, domain.MaxFileSize)
+		return c.Render(http.StatusInternalServerError, render.JSON(UploadResponse{
```

Comment:
Should the http status code here match the one two lines down?

Reply:
yes, fixed